### PR TITLE
SPUI-702 > Add warning popup

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: plugins/simple_kit/lib/simple_kit.dart [hunk 1]
```diff
@@ -194,3 +194,6 @@ export 'src/theme/provider/simple_theme_pod.dart';
 ///Countries
```
Comment (eli1stark) [inline plugins/simple_kit/lib/simple_kit.dart:194]: space between /// and Countries

File: plugins/simple_kit/lib/simple_kit.dart [hunk 2]
```diff
@@ -194,3 +194,6 @@ export 'src/theme/provider/simple_theme_pod.dart';
...
 export 'src/account/phone_number/simple_phone_numbers.dart';
 export 'src/account/phone_number/simple_number.dart';
+
+///Popup
```
Comment (eli1stark) [inline plugins/simple_kit/lib/simple_kit.dart:198]: space between

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import '../../../simple_kit.dart';
+
+void simpleShowWarningPopup(
+  BuildContext context, {
+  String? onTapCancelTitle,
+  Function(BuildContext builderContext)? onTapCancel,
+  String? subTitle,
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:9]: subtitle

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart [hunk 4]
```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import '../../../simple_kit.dart';
+
+void simpleShowWarningPopup(
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:5]: rename to sShowWarningPopup

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  BuildContext context, {
+  String? onTapCancelTitle,
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:7]: what is onTapCancelTitle?

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+      return Column(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          AlertDialog(
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.all(
+                Radius.circular(24.r),
+              ),
+            ),
+            buttonPadding: EdgeInsets.symmetric(horizontal: 20.w),
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:27]: comma

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+            insetPadding: EdgeInsets.symmetric(horizontal: 24.w),
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:28]: comma

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+  required String onTapTitle,
+  required Function(BuildContext builderContext) onTap,
+}) {
+  showDialog(
+    context: context,
+    builder: (builderContext) {
+      return Column(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          AlertDialog(
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:21]: use Custom widget instead of alertDialog

**Review by eli1stark — CHANGES_REQUESTED** [review]
Change

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+                  height: 80.h,
+                  child: Image.asset(asset),
+                ),
+              ],
+            ),
+            content: Padding(
+              padding: EdgeInsets.symmetric(horizontal: 20.w),
+              child: Column(
+                children: [
+                  SizedBox(
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:47]: remove SizedBox

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: plugins/simple_kit/lib/simple_kit.dart (hunk 2 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/simple_kit.dart:198]: done

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart (hunk 3 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart]: rename on subTitle -> secondaryText, onTapCancelTitle -> secondaryButtonName, onTapCancel -> onSecondaryButtonTap

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart (hunk 4 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:5]: done

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart (hunk 5 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart]: remove

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart (hunk 6 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:27]: done

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart (hunk 7 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:28]: done

File: plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart (hunk 9 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/shared/components/simple_show_warning_popup.dart:47]: done

**Pull request merged**